config.js

```diff
@@ -2,12 +2,5 @@ module.exports = {
     /**
      * The URL of the Jitsi Meet deployment that will be used.
      */
-    jitsiMeetURL: "https://meet.jit.si/",
-
-    /**
-     * If true, the authorization of remote control will be handled by jitsi
-     * meet electron app. Otherwise the authorization will be handled in Jitsi
-     * Meet.
-     */
-    handleRemoteControlAuthorization: false
```

Comment:
Because the next thing we have to do is to make sure that the user is sharing only entire screen (not an app window or tab). This will change the message of the dialog and we'll have to pass more arguments. I felt that actually supporting two equivalent windows (here and in meet) is becoming hard to maintain. 

Also I'm not sure if anybody will want to use this dialog ever and we need to make it prettier probably. 

That's why I decided to remove it for now and at some point if we need such a dialog we can revert it.